src/components/Sing/SingEditor.vue

```diff
@@ -42,7 +61,22 @@ const props = defineProps<{
 }>();
 
 const store = useStore();
-//const $q = useQuasar();
+
+const isSidebarOpen = computed(() => store.state.isSongSidebarOpen);
+const sidebarWidth = ref(300);
+let previousSidebarWidth = 300;
+
+// 開き直すときに最小幅が使われないようにするワークアラウンド。
+watch(
+  () => isSidebarOpen.value,
+  (isSidebarOpened) => {
+    if (isSidebarOpened) {
+      sidebarWidth.value = previousSidebarWidth;
+    } else {
+      previousSidebarWidth = sidebarWidth.value;
+    }
+  },
+);
```

Comment:
なるほどです！
これsetするときもisOpenか調べれば良いかも･･･？

updateをこうして
```ts
      @update:modelValue="
        (e) => {
          if (isSidebarOpen) sidebarWidth = e;
        }
      "
```

ここはこう。
```suggestion
const isSidebarOpen = computed(() => store.state.isSongSidebarOpen);
const sidebarWidth = ref(300);
```

なんか`isSidebarOpen`とupdateどっちが先に更新されるのか不安ですが、依存関係的にたぶん`isSidebarOpen`が更新された後に`update`が実行される･･･はず？

Reply:
似たような事を前に試してたんですが、多分QSplitter内部でも値を持っててそれを代入してくる感じの挙動をしていました。

Reply:
おっとなるほどです。
よくわかってないのですが、↑のコードは正しく動いた気がしたので提案してみました 🙏

Reply:
お、動きました、ありがとうございます